pom.xml

```diff
@@ -37,25 +35,25 @@
     <dependency> <!-- TODO switch to MockAuthorizationStrategy -->
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-auth</artifactId>
-      <version>2.2</version>
+      <version>3.1.7</version>
```

Comment:
needed newer versions for the baseline to avoid upper bounds issues - picked the most recent release,